packages/components/src/custom-select-control/index.tsx

```diff
@@ -149,16 +149,16 @@ function CustomSelectControl< T extends CustomSelectOption >(
 			);
 		} );
 
-	const { value: currentValue } = store.getState();
+	const { value: currentValue } = Ariakit.useStoreState( store );
```

Comment:
You can also access a specific value from the state by passing a second argument. However, I'm not sure if it makes any difference performance-wise.

```js
const currentValue = Ariakit.useStoreState( store, 'value' );
```

Reply:
Thanks for the suggestion @Mamaduka. It looks more readable for sure 🙌.